include/aikido/planner/parabolic/ParabolicTimer.hpp

```diff
@@ -43,6 +44,28 @@ std::unique_ptr<aikido::trajectory::Spline> computeParabolicTiming(
 std::unique_ptr<aikido::trajectory::Spline> convertToSpline(
     const aikido::trajectory::Interpolated& _inputTrajectory);
 
+/// Class for performing parabolic retiming on trajectories.
+class RetimeTrajectoryPostProcessor
+    : public aikido::planner::TrajectoryPostProcessor
+{
+public:
+  /// \param _velocityLimits maximum velocity for each dimension.
+  /// \param _accelerationLimits maximum acceleration for each dimension.
+  RetimeTrajectoryPostProcessor(
+      const Eigen::VectorXd& _velocityLimits,
+      const Eigen::VectorXd& _accelerationLimits);
+
+  /// Performs parabolic retiming on an input trajectory.
+  /// Documentation inherited.
```

Comment:
I think since you're adding something here, this might need to look like

```
/// Performs parabolic retiming on an input trajectory.
/// \copydoc TrajectoryPostProcessor::postprocess
```